cmd/minishift/cmd/start.go

```diff
@@ -187,13 +189,21 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
+	automountHostfolders(host.Driver)
```

Comment:
sure, I think it is the right place. This was just a general comment and not a call for action, unless you want to do a full out refactoring ;-)